addons/account/tests/test_account_all_l10n.py

```diff
@@ -16,6 +17,9 @@ def test_all_l10n(env):
     # Ensure the presence of demo data, to see if they can be correctly installed
     assert env.ref('base.module_account').demo, "Need the demo to test with data"
 
+    # Ensure that there are no modules marked to install
+    odoo.modules.reset_modules_state(env.cr.dbname)
```

Comment:
Try r+ again?

Reply:
Could you check which standalone test contaminate the db? It makes more sence to me to `reset_module_state` in that test.

Reply:
@yelizariev can you elaborate here and in the commit message why is resetting the module states required inside that test? My feeling is that you added this function call as an attempt to make the test passes but without real understanding why some modules were to-install/to-remove/to-update and without challenging whether resetting the states was correct.

Reply:
I really don't know what is going on runbot with those standalone tests. I was thinking that you guys have some ideas,

The build starts Odoo and runs `test_all_l10n` and fails immediately.

https://runbot.odoo.com/runbot/build/49846664
http://runbot137.odoo.com/runbot/static/build/49846664-master/logs/l10n_standalone.txt

May I get some insights how those build work? It gets DB dump from another build, right? Which build?

Also, I tried to restore following DB locally

![image](https://github.com/odoo/odoo/assets/186131/d15a795d-927c-4a40-b79a-d5686dab076b)

but modules states are fine

![image](https://github.com/odoo/odoo/assets/186131/2b645c0d-520c-48bf-8d5b-39ed7cd1d1dc)